demo/csi/digitalocean/plugin.nomad

```diff
@@ -28,8 +41,9 @@ job "digitalocean" {
       }
 
       resources {
-        cpu    = 500
-        memory = 256
+        cpu    = 100
+        memory = 64
+        memory_max = 128                                                                                       # be aware of using Nomad Scheduler Memory Oversubscription
```

Comment:
`nomad fmt` this file please. Not sure adding oversubscription improves this jobspec as a demonstration, either. Can you explain why you added it.